index.html

```diff
@@ -0,0 +1,89 @@
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Gabriela's Portafolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+    <header>
+      <nav class="top-navigation">
+        <a href="#" class="logo-placeholder">GabrielaSE</a>
+        <input type="checkbox" id="checkbox-icon-menu" name="checkbox-icon-menu">
+        <div class="normal-button">
+          <div class="icon-menu">
+            <div class="union">
+              <div class="vector vector-bar-1"></div>
+              <div class="vector vector-bar-2"></div>
+              <div class="vector vector-bar-3"></div>
+            </div>
+          </div>
+        </div>
+        <ul class="item-navigation">
+          <li class="nav-item">Portafolio</li>
+          <li class="nav-item">About</li>
+          <li class="nav-item">Contact</li>
+        </ul>
+        <div class="normal-button">
+          <div class="icon-mail">
+            <div class="union">
+              <span>
+                <svg id="icon-mail" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+                	<path d="M49.81,15.24H82.48a15.67,15.67,0,0,1,13.26,6.47,14.31,14.31,0,0,1,3,8.49c.08,6.24,0,12.49,0,18.73,0,7.95,0,15.9,0,23.85a15.48,15.48,0,0,1-10.65,15,17.39,17.39,0,0,1-5.55.87q-32.78.09-65.56,0C8.62,88.7,2.23,83.23,1,75.06a15.73,15.73,0,0,1-.14-2.39q0-20.69,0-41.38a15.64,15.64,0,0,1,12.1-15.58,21,21,0,0,1,4.54-.45C28.24,15.23,39,15.24,49.81,15.24Zm0,68.9H82.37c7.18,0,11.85-4.68,11.85-11.88q0-20.26,0-40.51c0-7.27-4.65-11.93-11.91-11.93h-65C10,19.82,5.4,24.48,5.39,31.76q0,20.26,0,40.51a18,18,0,0,0,.09,2.07,11.11,11.11,0,0,0,11.33,9.8C27.81,84.16,38.81,84.14,49.81,84.14Z"/>
+                	<path d="M60.27,57,52,65c-1.66,1.61-2.74,1.62-4.38,0l-8.19-8-5.3-5.17q-10.26-10-20.5-20a11.5,11.5,0,0,1-1-1,2.09,2.09,0,0,1,.1-2.78,2.23,2.23,0,0,1,2.93-.56,8.86,8.86,0,0,1,1.24,1.05L48.69,59.53l1.1,1c.39-.36.74-.66,1.07-1l31.84-31a10.62,10.62,0,0,1,.89-.8,2.21,2.21,0,0,1,3.21.26,2.09,2.09,0,0,1-.06,3.11c-1.52,1.56-3.09,3.07-4.65,4.59Z"/>
+                </svg>
+              </span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="page-section headline-section">
+      <div class="content-text primary-text">
+        Hey There. I’m Gabriela
+      </div>
```

Comment:
- [ ] Here just one small adjustment this should be a `h1`.
something like this.
```suggestion
      <h1 class="content-text primary-text">
        Hey There. I’m Gabriela
      </h1>
```
_tip: you can just apply this suggestion 😉_